jdi-light-html-tests/src/main/java/io/github/com/entities/UserInfo.java

```diff
@@ -3,8 +3,5 @@
 import com.epam.jdi.tools.DataClass;
 
 public class UserInfo extends DataClass<UserInfo> {
-    public String name;
-    public String phone;
-    public String email;
-    public String city;
+    public String name, phone, email, city;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5478709)